lib/renderer.js

```diff
@@ -127,6 +127,32 @@ function applyDirectives(text) {
   return text;
 }
 
+function addHeadingFragments(text) {
+  if (configuration.getConfig().features.fragments) {
```

Comment:
For better readibility I always prefer to shortcut ifs, like `if (!something) { return }`. In this case I will "fix it" after the merge :)